src/vivarium_public_health/risks/effect.py

```diff
@@ -37,7 +37,7 @@ class RiskEffect(Component):
     .. code-block:: yaml
 
        configuration:
-           effect_of_risk_on_affected_risk:
```

Comment:
Changes in this file have already been merged into the release candidate so once Rajan merges they will not be here.